deploy/crds/org_v1_che_crd.yaml

```diff
@@ -271,7 +271,9 @@ spec:
                   type: string
                 tlsSecretName:
                   description: Name of a secret that will be used to setup ingress
-                    TLS termination if TLS is enabled. See also the `tlsSupport` field.
+                    TLS termination if TLS is enabled. If the field is empty string,
```

Comment:
Sorry, but where is changes in the che_types? I think you should change this text in the https://github.com/eclipse/che-operator/blob/master/pkg/apis/org/v1/che_types.go and then regenerated crd using https://github.com/eclipse/che-operator/blob/master/olm/update-crd-files.sh or you can use VSCode task https://github.com/eclipse/che-operator/blob/master/.vscode/tasks.json#L5

Reply:
Right, hank you. I'll make corresponding changes.